### ISSUE TITLE
Customized Notification

### ISSUE THREAD

**Issue by miferber** [Falcosc/locus-addon-tasker#5]
I have a another use case. Here I'm not sure if Tasker/Locus Map would already support this.

I always use Locus Map with activated OFF-Track notification (Abweichungs-Alarm). While hiking I can keep the display to be off (power saving) and I need only to take a look at the map/display if I get an OFF-Track notification.

The request: Is it possible to generate a "customized" notification  and send this to a Wear OS Watch for example (i.e. "Your are OFF-track" & Vibration via WearTasker).
If this would be working I could keep the mobile in my backpack (Display & sound Off) all the time by just checking the notifications on the watch (and monitoring the track with the Locus Wear App).

**Comment by Falcosc** [comment]
I know that this is not possible over the request based API. I have to think about how to create this idea at locus Helpdesk because it's a little bit different to receive events from locus instead of using the API to ask Locus Maps for specific data.

**Comment by miferber** [comment]
Yes, that would be great if your guess is hopefully possible to be implemented.

**Comment by Falcosc** [comment]
@miferber which notifications are synced between your smartphone and your watch?
I did read about the problem of receiving only notifications from certain apps on android wear because not all notifications are synced between smartphone and smartwatch.

Here is the idea to get this working: https://help.locusmap.eu/topic/locus-for-wear-out-of-route-notification

In case if locus notifications can't be synced between smartphone and smartwatch, are notifications created by tasker synced to your watch? Please try out if you can use tasker on your smartphone to get watch notifications.
If tasker could send you notifications I could re-implement the whole "out of route" functionality if you like to spend some computing-time/battery-life from your smartphone to compare a lot of track points with current GPS position every 5 or 10 seconds.

**Comment by Falcosc** [comment]
@miferber did you try to set "vibrate" on "out of route" notification? Maybe this event gets syncroniced to android wear?

**Comment by miferber** [comment]
I enabled "vibrate" & "sound" notification in locus but these notifications are not shown on Wear OS. Anyway, I have installed "WearTasker" on my watch. This App appears as a Plug-in in Tasker on the smsrtphone and it supports to create notifications which are shown on the watch. So I think this is no big issue.
 But I don't know how to "receive" a notification (Off Track notification) in Tasker from Locus .

**Comment by Falcosc** [comment]
In best case a notification will be implemented by locus team if it will get enough votes.
Normally all notifications are synced between smartphone and watch.

If you have problem with notification sync you could listen for notification events from Tasker and refire a notification event to your watch. But we need to wait for the 3rd out of route option "notification"
I don't recreate the whole out of route feature to trigger unless locus team will deny the idea.

**Comment by Falcosc** [comment]
@miferber don't forget to vote for locus ideas, like our new one: https://help.locusmap.eu/topic/locus-for-wear-out-of-route-notification

**Comment by Falcosc** [comment]
@miferber if you are still interested, you may ask for news in the https://help.locusmap.eu/topic/locus-for-wear-out-of-route-notification topic. Maybe they already have anything implemented which will be synced with your watch, like notifications.

Did you try the current navigation implementation? They use notifications for navigation instructions, maybe there is an out of route notification in the navigation mode.
